Unable to start activity ComponentInfo{app.lawnchair/com.android.launcher3.dragndrop.AddItemActivity}

```
Lawnchair bug report 2023年7月7日 00:24:24
version: 12.1.0 (a32d580) (12010005)
commit: a32d580
build.brand: motorola
build.device: rtwo
build.display: T1TR33.4-30-62
build.fingerprint: motorola/rtwo_cn/rtwo:13/T1TR33.4-30-62/d90c0-e6e07:user/release-keys
build.hardware: qcom
build.id: T1TR33.4-30-62
build.manufacturer: motorola
build.model: XT2301-5
build.product: rtwo_cn
build.type: user
version.codename: REL
version.incremental: d90c0-e6e07
version.release: 13
version.sdk_int: 33
display.density_dpi: 446
isRecentsEnabled: false

error: Uncaught exception

java.lang.RuntimeException: Unable to start activity ComponentInfo{app.lawnchair/com.android.launcher3.dragndrop.AddItemActivity}: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
 at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3772)
 at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3920)
 at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:101)
 at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:135)
 at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:95)
 at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2393)
 at android.os.Handler.dispatchMessage(Handler.java:111)
 at android.os.Looper.loopOnce(Looper.java:238)
 at android.os.Looper.loop(Looper.java:357)
 at android.app.ActivityThread.main(ActivityThread.java:8118)
 at java.lang.reflect.Method.invoke(Native Method)
 at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
 at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:957)
Caused by: java.lang.IllegalStateException: You need to use a Theme.AppCompat theme (or descendant) with this activity.
 at g.j0.x(Unknown Source:747)
 at g.j0.j(Unknown Source:0)
 at androidx.appcompat.app.a.setContentView(SourceFile:2)
 at com.android.launcher3.dragndrop.AddItemActivity.onCreate(Unknown Source:43)
 at android.app.Activity.performCreate(Activity.java:8479)
 at android.app.Activity.performCreate(Activity.java:8458)
 at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1386)
 at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:3753)
 ... 12 more
```

--- a/lawnchair/res/values/themes.xml
+++ b/lawnchair/res/values/themes.xml
@@ -31,4 +31,14 @@
         <item name="android:enforceNavigationBarContrast">false</item>
         <item name="windowNoTitle">true</item>
     </style>
+
+    <style name="Theme.AppCompat.Translucent.NoTitleBar" parent="Theme.AppCompat.NoActionBar">
+        <item name="android:windowBackground">@android:color/transparent</item>
+        <item name="android:colorBackgroundCacheHint">@null</item>
+        <item name="android:windowIsTranslucent">true</item>
+        <!-- Note that we use the base animation style here (that is no
+                 animations) because we really have no idea how this kind of
+                 activity will be used. -->
+        <item name="android:windowAnimationStyle">@android:style/Animation</item>
+    </style>
 </resources>

--- a/res/values-night/styles.xml
+++ b/res/values-night/styles.xml
@@ -18,7 +18,7 @@
 -->
 
 <resources>
-    <style name="AddItemActivityTheme" parent="@android:style/Theme.Translucent.NoTitleBar">
+    <style name="AddItemActivityTheme" parent="Theme.AppCompat.Translucent.NoTitleBar">
         <item name="widgetsTheme">@style/WidgetContainerTheme.Dark</item>
         <item name="android:windowTranslucentStatus">true</item>
     </style>

--- a/res/values/styles.xml
+++ b/res/values/styles.xml
@@ -313,7 +313,7 @@
         <item name="android:stateListAnimator">@null</item>
     </style>
 
-    <style name="AddItemActivityTheme" parent="@android:style/Theme.Translucent.NoTitleBar">
+    <style name="AddItemActivityTheme" parent="Theme.AppCompat.Translucent.NoTitleBar">
         <item name="widgetsTheme">@style/WidgetContainerTheme</item>
         <item name="android:windowLightStatusBar">true</item>
         <item name="android:windowTranslucentStatus">true</item>

